### DISCUSSION TITLE
Where is the source code for the upcoming 3.0 version?

### DISCUSSION THREAD

**Discussion by linucksrox** [requarks/wiki#4964] [Help / Questions] [answered]
I would like to test and possibly contribute to development of the 3.0 version because some features are planned that I would like to utilize and that might allow us to migrate our old Drupal 7 knowledgebase to Wiki.js. Any way I can build from source and see where things are at currently? All I see on Github is the latest 2.x version.

**Comment by Phil-hostwiki** [comment, marked as answer]
V3 repository: https://github.com/Requarks/wiki-vnext.

**Reply by linucksrox** [reply]
Thanks! I could not find it via searching and didn't think to browse the other Requarks repositories.
